hadoop-ozone/ozone-manager/src/main/resources/webapps/ozoneManager/ozoneManager.js

```diff
@@ -113,5 +113,12 @@
         require: {
             overview: "^overview"
         },
+        controller: function ($http) {
+            var ctrl = this;
+            $http.get("jmx?qry=Ratis:service=RaftServer,group=*,id=*")
+                .then(function (result) {
```

Comment:
can remove this part if the above approach works

Reply:
We will be needing it to display the group ID and the current role !!